src/server/pfs/server/api_server.go

```diff
@@ -788,14 +789,110 @@ func (a *apiServer) RunLoadTest(ctx context.Context, req *pfs.RunLoadTestRequest
 	return resp, nil
 }
 
-func (a *apiServer) runLoadTest(pachClient *client.APIClient, branch *pfs.Branch, specBytes []byte, seed int64) error {
+func (a *apiServer) runLoadTest(pachClient *client.APIClient, branch *pfs.Branch, specStr string, seed int64) error {
 	spec := &pfsload.CommitsSpec{}
-	if err := yaml.UnmarshalStrict(specBytes, spec); err != nil {
+	if err := yaml.UnmarshalStrict([]byte(specStr), spec); err != nil {
 		return err
 	}
 	return pfsload.Commits(pachClient, branch.Repo.Name, branch.Name, spec, seed)
 }
 
+func (a *apiServer) RunLoadTestDefault(ctx context.Context, _ *types.Empty) (resp *pfs.RunLoadTestResponse, retErr error) {
+	func() { a.Log(nil, nil, nil, 0) }()
+	defer func(start time.Time) { a.Log(nil, nil, retErr, time.Since(start)) }(time.Now())
+	for _, spec := range defaultLoadSpecs {
+		var err error
+		resp, err = a.RunLoadTest(ctx, &pfs.RunLoadTestRequest{
+			Spec: spec,
+		})
+		if err != nil {
+			return nil, err
+		}
+		if resp.Error != "" {
+			return resp, nil
+		}
+	}
+	return resp, nil
+}
+
+var defaultLoadSpecs = []string{`
```

Comment:
If we are going to hard code these I think we should store them structured. Like call must parse on them or something.

Reply:
Unless they are exercised in tests.  e.g. All the SQL we have is going to be valid if it's sent to the database at all during tests.

Reply:
Would that be fine in a followup? I would like to get this in ASAP so we can have regression testing in Hub.